tests/vagrant/templates/Vagrantfile.tmpl

```diff
@@ -24,7 +24,7 @@ Vagrant.configure("2") do |config|
 
     config.vm.define :"node02" do |node|
         node.vm.hostname = "node02"
-        node.vm.network "forwarded_port", guest: 1337, host: 1338, host_ip: "*"
+        node.vm.network "forwarded_port", guest: 1337, host: 1338
```

Comment:
and this?

Reply:
isn't that something you changed recently? I may still need to rebase this on top of main.

Reply:
yes... so you need to rebase then